templates/post_form.html

```diff
@@ -89,6 +89,20 @@
 				<script>load_captcha("{{ config.captcha.provider_get }}", "{{ config.captcha.extra }}");</script>
 			</td>
 		</tr>
+		<!-- New thread capt -->
+		{% elseif config.new_thread_capt %}
+			<!-- :^) -->
+			{% if id %} 
```

Comment:
Because I have no idea what {% this language %} is and didn't know I could do that :^)

Just changed it to "if not".